Google Play Music/JSBound.cs

```diff
@@ -95,14 +95,11 @@ public void goBig()
         // Fired from javascript when a different song has been playing for over half it's duration. (As according to last.fm's scrobbling spec)
         public void songScrobbleRequest(string song, string artist, string album, int timestamp)
         {
-            try
+            // Check last.fm user authenticated before attempting to "scrobble" the current song.
+            if (LastFM.user_key != null)
             {
                 new LastFM().scrobbleTrack(artist, song, timestamp).Wait();
             }
-            catch (Exception e)
```

Comment:
Ok, I didn't know the use case since I don't use LastFM. I guess I need to sign up for it and work on some of these use cases.